framework/contracts/native/version-control/src/contract.rs

```diff
@@ -89,6 +89,11 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg) -> V
             namespace,
             monetization,
         } => set_module_monetization(deps, info, module_name, namespace, monetization),
+        ExecuteMsg::SetModuleInstantiationFunds {
```

Comment:
I was thinking to merge the metadata and this method into a single endpoint to configure the module

Reply:
> I was thinking to merge the metadata and this method into a single endpoint to configure the module

And monetization?

Reply:
Yeah if possible! And have it be a `non_exhaustive`